e2e/__tests__/globalSetup.test.ts

```diff
@@ -27,6 +27,7 @@ const customTransformDIR = path.join(
 const nodeModulesDIR = path.join(tmpdir(), 'jest-global-setup-node-modules');
 const rejectionDir = path.join(tmpdir(), 'jest-global-setup-rejection');
 const e2eDir = path.resolve(__dirname, '../global-setup');
+const esmTmpDir = path.join(tmpdir(), 'jest-global-setup-esm');
```

Comment:
is this (and the one in teardown) used?

Reply:
Yes, the directory  is created in [setup file](https://github.com/WeiAnAn/jest/blob/884c7e0a1653e339a26c568d35814ea1b83fd9b1/e2e/global-setup-esm/setup.js#L15-L24).

Reply:
sure, but what is that directory used for? In the test it looks like it's created and deleted but never used

Reply:
GlobalSetup/Teardown will read this directory and do some assertion.
If we don't clean directory before testing, it will failed in second round.

Reply:
even after [`884c7e0` (#11232)](https://github.com/facebook/jest/pull/11232/commits/884c7e0a1653e339a26c568d35814ea1b83fd9b1)?

Reply:
Yes.
It fix #11267 globalSetup/globalTeardown e2e test.

Reply:
Aha, gotcha!